compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaLexer.java

```diff
@@ -546,6 +546,8 @@ private STToken processIdentifierOrKeyword() {
                 return getSyntaxToken(SyntaxKind.ELSE_KEYWORD);
             case LexerTerminals.WHILE:
                 return getSyntaxToken(SyntaxKind.WHILE_KEYWORD);
+			case LexerTerminals.PANIC:
```

Comment:
Not formatted correctly.

Reply:
sorry, my editor used tabs,so it looked fine in mine, I ll fix this.